<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lunes">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>25</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Garamond</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Lunes</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_lunes">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>401</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="martes">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>155</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Garamond</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Martes</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_martes">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>401</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="miercoles">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>285</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Garamond</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Miercoles</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_miercoles">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>401</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="jueves">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>415</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Garamond</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Jueves</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_jueves">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>450</y>
     <width>401</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="viernes">
   <property name="geometry">
    <rect>
     <x>596</x>
     <y>25</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Garamond</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Viernes</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_viernes">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>60</y>
     <width>401</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="sabado">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>160</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Garamond</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Sabado</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sabado">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>190</y>
     <width>401</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="domingo">
   <property name="geometry">
    <rect>
     <x>596</x>
     <y>285</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Garamond</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Domingo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_domingo">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>320</y>
     <width>401</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_horario">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>530</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
